rls/src/main/java/io/grpc/rls/CachingRlsLbClient.java

```diff
@@ -137,6 +152,23 @@ private CachingRlsLbClient(Builder builder) {
             childLbHelperProvider, new BackoffRefreshListener());
   }
 
+  private static boolean isOobChannelDirectpathEnabled() {
```

Comment:
You probably want to make system property more "static" to your program. That is, the system property should be read when your code is loaded instead of when this method is executed.

Reply:
done. PTAL